src/Service.Host/client/src/components/importBooks/ImpBookPrintOut.js

```diff
@@ -60,6 +61,10 @@ function ImpBookPrintOut({
         return date ? moment(date).format('DD-MMM-YYYY') : '-';
```

Comment:
would probably rename to toDdMMYYYY(date) or something

Reply:
Yeah I'll rename that haha can probably do better than getNicerDate